Detections/PulseSecureVPN/LargeNumberofDistinctFailedUserLogins.yaml

```diff
@@ -0,0 +1,29 @@
+id: 1fa1528e-f746-4794-8a41-14827f4cb798
+name: Large Number of Distinct Failed User Logins
+description: |
+  'This creates an incident in the event a Pulse Secure VPN server experiences failed login attempts from a large number of distinct users.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - PulseSecureVPN
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - Discovery 
+relevantTechniques:
+  - T1110
+  - T1201 
+  
+query: |
+
+  let timeframe = ago(1h);
+  PulseSecureVPN
+  | where TimeGenerated >= timeframe
+  | where Messages startswith "Login failed"
+  | summarize dcount(User) by Computer, bin(TimeGenerated, 15m)
+  | where dcount_User > 100
+  | extend HostCustomEntity = Computer
```

Comment:
For use as a Detection, we will need to bring the Time value thru.  In this case -
| extend timestamp = bin_TimeGenerated, HostCustomEntity = Computer

Reply:
Corrections made and pushed